airflow/providers/cncf/kubernetes/utils/pod_launcher.py

```diff
@@ -51,10 +54,23 @@ def should_retry_start_pod(exception: Exception) -> bool:
 class PodStatus:
     """Status of the PODs"""
 
-    PENDING = 'pending'
-    RUNNING = 'running'
-    FAILED = 'failed'
-    SUCCEEDED = 'succeeded'
+    PENDING = 'Pending'
+    RUNNING = 'Running'
+    FAILED = 'Failed'
+    SUCCEEDED = 'Succeeded'
+
+    terminal_states = {FAILED, SUCCEEDED}
```

Comment:
We should leave this change to a separate PR or change https://github.com/apache/airflow/blob/main/airflow/kubernetes/pod_launcher_deprecated.py#L54-L60 too

Reply:
is that because you think that someone might accidentally use the deprecated enum with the current launcher?

do you know what purpose the deprecated launcher serves and when we can remove it?  i think @jedcunningham was thinking we do a major release for this so we can flip the default behavior of "delete pod" to true.  if so, would we also take that opportunity to remove the deprecated pod launcher?

Reply:
Unfortunately that's in core, not the provider. I'm inclined to leave it alone (I don't think it'd break anything?).